client/state/agency-dashboard/package.json

```diff
@@ -0,0 +1,5 @@
+{
+	"sideEffects": [
```

Comment:
Out of curiosity, why do we need that `sideEffects` here?

Reply:
`init.js` doesn't have any exports which effectively means that this is not used anywhere in the application for Webpack. So, it needs to be added to the side effect list so it will not be unintentionally dropped during tree shaking.